test/test_tensor_creation_ops.py

```diff
@@ -832,6 +832,111 @@ def compare_helper_(like_fn, t):
                     tp = t.permute(p)
                     compare_helper_(like_fn, tp)
 
+    @onlyOnCPUAndCUDA
+    @dtypes(*torch.testing.get_all_dtypes(include_bfloat16=False, include_bool=False))
+    def test_hsplit(self, device, dtype):
+        inputs = (
+            ((), 3),
+            ((), [2, 4, 6]),
+            ((6), 2),
+            ((6), 4),
+            ((6), [2, 5]),
+            ((6), [7, 9]),
+            ((3, 8), 4),
+            ((3, 8), 5),
+            ((3, 8), [1, 5]),
+            ((3, 8), [3, 8]),
+            ((S, S, S), 2),
+            ((S, S, S), [1, 4]),
+            ((S, 0, S), 3),
+            ((S, S, 0), [2, 6]),
+        )
+
+        for shape, arg in inputs:
+            t = make_tensor(shape, dtype, device)
+            t_n = t.cpu().numpy()
```

Comment:
t_n -> t_np would be more readible

Reply:
This is the right idea.

`t` for a single tensor, `a`, `b`... for multiple tensors

When working with NumPy then `t` and `a` are appropriate when there's a single tensor and corresponding array, alternatively `a`, `b`, ... and `a_np`, `b_np`... 

Sometimes a more specific name, like `eigenvalues` is appropriate, and in that case the corresponding NumPy array should also be `eigenvalues_np`